context/FilterConsultantsContext.js

```diff
@@ -0,0 +1,87 @@
+import React, { createContext, useContext, useState } from 'react';
+import defaultValues from '../defaultdata/DefaultFilterSettings.json';
+
+// Create a context
+const SpecialitiesContext = createContext();
```

Comment:
Nitpick on the naming: I suggest that the context and the provider component names match that of the file, and reflect all the filters instead of just the specialties. Again, this is for code readability

Reply:
This was a pretty big deal thanks for pointing it out. I actually remember doing this and telling myself I would change it later. Just fixed the file name too ConsultantFilterContext. I then created the context with ConsultantFiltersContext, then created the function ConsultantFiltersProvider. Finally we have the exported version as useConsultantFilters. Let me know if this sounds okay.